exporter/elasticsearchexporter/trace_exporter.go

```diff
@@ -122,5 +124,5 @@ func (e *elasticsearchTracesExporter) pushTraceRecord(ctx context.Context, resou
 	if err != nil {
 		return fmt.Errorf("Failed to encode trace record: %w", err)
 	}
-	return pushDocuments(ctx, e.logger, fIndex, document, e.bulkIndexer, e.maxAttempts)
+	return pushDocuments(ctx, e.logger, fIndex, document, e.bulkIndexer, e.maxAttempts, e.retryOnStatus)
```

Comment:
Everything looks good, BTW, the`pushDocuments` function is loaded with too many arguments now. Hopefully, it will shrink  in the future :-P.